.ci/run_metrics_PR_ci.sh

```diff
@@ -102,6 +94,8 @@ check() {
 				sudo mkdir -p ${CHECKMETRICS_CONFIG_DEFDIR}
 				sudo cp ${CM_DEFAULT_DENSITY_CONFIG} ${CM_BASE_FILE}
 			fi
+		elif [ -n "${METRICS_CI_CLH}" ]; then
```

Comment:
What about `${METRICS_CI_CLH_BAREMETAL}` ? So we can easily identify more profiles in  in the future.